packages/components/pages/f-checkout/src/store/tests/checkout.module.test.js

```diff
@@ -20,7 +20,11 @@ import {
     mockAuthToken, mockAuthTokenNoNumbers, mockAuthTokenNoMobileNumber
 } from '../../components/_tests/helpers/setup';
 import {
-    VUEX_CHECKOUT_ANALYTICS_MODULE, DEFAULT_CHECKOUT_ISSUE, ERROR_TYPES, DUPLICATE_ORDER
+    VUEX_CHECKOUT_ANALYTICS_MODULE,
+    DEFAULT_CHECKOUT_ISSUE,
+    ERROR_TYPES, DUPLICATE_ORDER,
```

Comment:
Nit: for consistency
```suggestion
    DUPLICATE_ORDER,
    ERROR_TYPES,
```

Reply:
For someone with serious OCD around formatting, I'm shocked that I missed this. All done now.